docs/contributor/04-10-workflows.md

```diff
@@ -95,3 +95,59 @@ The workflow performs the following actions:
 - Waits for the binary image to be ready in the registry
 - Installs the module
 - Runs the E2E BTP Manager secret customization test on the cluster
+
+### Performance Tests
+
+The [workflow](/.github/workflows/run-performance-tests-reusable.yaml) runs performance and tests on the k3s cluster. The following parameters are required from the calling workflow:
+
+| Parameter name       | Required | Description                     |
+|----------------------|----------|---------------------------------|
+| **image-repo**       | yes      | Binary image registry reference |
+| **image-tag**        | yes      | Binary image tag                |
+| **credentials-mode** | yes      | Specifies whether to use real or dummy credentials |
+
+The workflow performs the following actions for all jobs:
+- Prepares the k3s cluster with the Docker registry
+- Waits for the binary image to be ready in the registry
+- Installs the module
+
+#### Jobs
+1. **Frequent Secret Update Test**
+- **Purpose**: Evaluates the system's response time and reconciliation success rate when the `sap-btp-manager` secret is updated frequently.
+- **Steps**:
+    - Patches the `sap-btp-manager` secret in a loop to simulate frequent updates.
+    - Fetches metrics from the `btp-manager-controller-manager` to measure average reconcile time, reconcile errors, and other reconciliation statistics.
+- **Failure Handling**:
```

Comment:
`Failure Handling` implies some information on how to cope with the failure, which is not the case here. How about:
**The test fails in the following conditions**:
   - The average reconcile time exceeds the defined threshold.
   - Some reconcile errors are detected.

Reply:
You are right. I will change it to `The test fails in the following conditions`